LWA/LWA_bifrost.py

```diff
@@ -866,14 +866,14 @@ def main(self):
                         # Y
                         a = self.antennas[2 * i + 1]
                         delay = a.cable.delay(freq) - a.stand.z / speedOfLight
-                        phases[:, :, i, 1, :, :] = numpy.exp(2j * numpy.pi * freq * delay)
-                        phases[:, :, i, 1, :, :] /= numpy.sqrt(a.cable.gain(freq))
-                    # Explicit bad and suspect antenna masking - this will
-                    # mask an entire stand if either pol is bad
-                    if self.antennas[2 * i + 0].getStatus() < 33 or self.antennas[2 * i + 1].getStatus() < 33:
-                        phases[:, :, i, :, :, :] = 0.0
-                    # Explicit outrigger masking - we probably want to do
-                    # away with this at some point
+                        phases[:, :, 1, i, :, :] = numpy.exp(2j * numpy.pi * freq * delay)
+                        phases[:, :, 1, i, :, :] /= numpy.sqrt(a.cable.gain(freq))
```

Comment:
Was this transpose of indices accidental? It doesn't look like it should have changed, and I get an index error when I try to run.

Reply:
Yeah, they are swapped.

Reply:
Oh, and there was another one as well a little further down.